src/ngraph_builder.cc

```diff
@@ -436,6 +436,86 @@ static Status TranslateBinaryOp(
       });
 }
 
+// Helper function for translating QuantizedAvgPool and QuantizedMaxPool
+static Status TranslateQuantizedPoolOp(
+    const Node* op, const std::vector<const Tensor*>& static_input_map,
+    Builder::OpMap& ng_op_map, std::string pooling_name) {
+  bool is_quantizedAvgPool = pooling_name == "QuantizedAvgPool" ? true : false;
+  bool is_quantizedMaxPool = pooling_name == "QuantizedMaxPool" ? true : false;
+
+  // Doesn't  compile
+  // bool is_quantizedAvgPool =
+  // std::is_same<T,ng::builder::ScaledQuantizedAvgPool>::value;
+  // bool is_quantizedMaxPool =
+  // std::is_same<T,ng::builder::ScaledQuantizedMaxPool>::value;
```

Comment:
we can remove these comments. `ScaledQuantizedAvgPool`, `ScaledQuantizedMaxPool ` are builder functions (and not classes like ng nodes), so we can't template them